gulpfile.js

```diff
@@ -19,7 +19,7 @@ function createDist() {
                 min: `.lite.${pkg.version}.min.js`
             }
         }))
-        .pipe(gulp.dest('bundles'));
+        .pipe(gulp.dest('assets/js'));
 
```

Comment:
will gulp automatically create the assets/js folder? right now the examples still have the bundles folder, but no assets/js folder. Are you planning to delete those from the examples later on and add the assets/js folder?